front/app/modules/commercial/customizable_navbar/admin/containers/HiddenNavbarItemList/HiddenNavbarItemList.test.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { render, screen, fireEvent } from 'utils/testUtils/rtl';
-import HiddenNavbarItemList from './HiddenNavbarItemList';
+import HiddenNavbarItemList from '.';
 import allNavbarItems from 'hooks/fixtures/navbarItems';
-import { addNavbarItem } from '../../services/navbar';
-import { deletePage } from 'services/pages';
+import { addNavbarItem } from '../../../services/navbar';
+import { deletePage } from '../../../services/pages';
```

Comment:
Nitpicky : When the absolute path is simple enough, I like it better than the relative one, makes it easier to move things around or to replace paths